.changeset/dirty-phones-report.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': major
+---
+
+breaking: stricten types around `depends` and `invalidate` to require a `:` when passing a string
```

Comment:
it seems "stricten" is a word, but I'd never heard it before today and Chrome's spell checker is telling me it isn't a word, so maybe safer to omit it since people might just think it's a typo
```suggestion
breaking: the types for `depends` and `invalidate` now require a `:` when passing a string
```